driver-core/src/main/com/mongodb/operation/CreateCollectionOperation.java

```diff
@@ -251,6 +335,11 @@ private BsonDocument asDocument() {
         if (indexOptionDefaults != null) {
             document.put("indexOptionDefaults", indexOptionDefaults);
         }
+        if (validator != null) {
```

Comment:
The server allows action and/or level to be specified even when validator is not, so our operation should to.

e.g.

> db.runCommand({create : 'validated8',  validationAction : "error" } )
{ "ok" : 1 }

We definitely should not silently ignore level and action like this.